doc/user/add-method.xml

```diff
@@ -89,31 +70,35 @@ int main() { printf("Hello, world!\n"); }
 
   <para>
 
-  As mentioned, a pseudo-builder also provides more flexibility
-  in parsing arguments than you can get with a &Builder;.
+  A pseudo-builder is useful because it provides more flexibility
+  in parsing arguments than you can get with a standard Builder method.
```

Comment:
Curious why de-entity Builder?
Also Builders' aren't always methods, for example in the DefaultEnvironment().. they are accessible  without the env.

Reply:
It's too awkward to keep going method-or-function all the time. Can of course do so.

Meanwhile, builders are things, didn't seem to me like they need special markup. Also not a big deal.

Reply:
I think it's valuable to markup builder as it's not the verb builder but rather the abstraction builder which  part of SCons.